src/modules/Home/components/Content/index.jsx

```diff
@@ -98,7 +104,7 @@ function Content() {
               className={classes.searchInput}
               variant="outlined"
               placeholder="Search"
-              onChange={(event) => setInputValue(event.target.value)}
+              onChange={(event) => setChipsArr({ search: event.target.value })}
```

Comment:
`onChange={(event) => setChipsArr(filterValue => { ...filterValue, search: event.target.value })}`

Reply:
You should modify only what you really need, not the whole state.

Reply:
applied the spread operator